advocacy_docs/edb-postgres-ai/databases/database_operations/resource_monitoring/cpu_monitoring.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Basic CPU monitoring 
+navTitle: CPU monitoring
+description: Perform basic monitoring of your CPU metrics. 
+---
+
+CPU usage is essential for the processing of queries, connections, transactions and background processes. Therefore, it is important to ensure your CPU configuration can cover the requirements of your applications, and it has enough buffer to continue working through workload peeks.
+
+## Healthy CPU 
+
+As a general rule, a database server is considered to have a healthy CPU configuration when the average **CPU** usage is stable and does not surpass the 80% threshold of the allotted CPU resources over a longer period of time. This ensures the database server isn't overburdened.
+
+The key determining factor is the stability of client queries. If you expect an exponential increase of client read and write operations, 80% could soon not be enough.
```

Comment:
'stability of client queries' is kind if cryptic. We should spell out that the number and complexity of queries being executed at a given time drives CPU usage.

Also, there's no reason to call out 'exponential' growth here. Any expected growth needs to be considered regardless of its nature (and exponential growth could never be accommodated anyway as by definition you'd keep growing faster and faster and exceed any capacity)